specifyweb/frontend/js_src/lib/components/Merging/Usages.tsx

```diff
@@ -16,32 +16,56 @@ export function UsagesSection({
 }: {
   readonly resources: RA<SpecifyResource<AnySchema>>;
 }): JSX.Element {
+  const [blockerLoaded, setBlockerLoaded] = React.useState<boolean>(false);
+  const handleBlockersLoaded = React.useCallback(
+    () => setBlockerLoaded(true),
+    [setBlockerLoaded]
+  );
```

Comment:
Hmm I could. But this is simple enough to not have that. I don't use useBooleanState unless I need toggle :)

Reply:
yes toggle callback is useful feature of useBooleanState
the other useful feature of useBooleanState is that when the default value you pass to it changes, the state is also changed (thanks to useTriggerState underneath)
but the reason I use useBooleanState most of the time is because it makes code more readable (in my opinion)
instead of having to do `onClick={(): void => setIsOpen(false) }`, I can just do `onClick={handleClose}`

though, useBooleanState would be slower than useState (as useBooleanState involves 3 useCallback calls, and one useLayoutEffect ) - though I never benchmarked so don't know if it makes a difference outside of extreme cases

still the benefits are not too strong, so I don't require people to use useBooleanState (but encourage when applicable)

Reply:
although, I am looking at this code again, and useBooleanState would totally make things simpler.

just think, you don't event need this:
```ts
  const [blockerLoaded, setBlockerLoaded] = useBooleanState();
	  const handleBlockersLoaded = React.useCallback(
	    () => setBlockerLoaded(),
	    [setBlockerLoaded]
	  );
```

because you can just do:
```ts
  const [blockerLoaded, setBlockerLoaded] = useBooleanState();
	  const handleBlockersLoaded = setBlockerLoaded;
```

and thus:

```ts
  const [blockerLoaded, handleBlockersLoaded] = useBooleanState();
```

tada 🎉

Reply:
fixed